### DISCUSSION TITLE
How to use the Lua bit module (library) ?

### DISCUSSION THREAD

**Discussion by verhagen** [nesbox/TIC-80#2532] [Q&A]
Hello TIC-80 developers

My question is: can the Lua **bit** module be used? And if so, how should it be done?

In pico-8 there is the bit '>>>' shift operator, which puts in zero from the left. This '>>>' operator is not available in TIC-80.

But some searching brought up this (standard) Lua module [BitwiseOperators](http://lua-users.org/wiki/BitwiseOperators).
As I've already used the `math` library, so I thought, this `bit` library, might be also available.

But when using `bit.blogic_rshift(n, bits)` there getting this error: `attempt to index a nil value (global 'bit')`
Also reading about importing [modules](http://lua-users.org/wiki/ModulesTutorial), but writing the `local bit=require("bit")` did not do the trick.

Thanks for your help

**Comment by Weeppiko** [comment]
TIC-80 does have bitwise operators, you're just writing it wrong, it is ">>" for right shift, and not ">>>".

**Reply by verhagen** [reply]
Indeed seams to be correct case.

In Java / C an 1 is inserted on the left, when using the `>>` operator and a zero is inserted when `>>>` is used.
